lib/utils/validateObjectWithArrayProps.js

```diff
@@ -3,41 +3,29 @@
 const { isPlainObject } = require('./validateTypes');
 
 /**
- * Check whether the variable is an object and all its properties are arrays of values
+ * Check whether the variable is an object and all its properties are one or more values
  * that satisfy the specified validator(s):
  *
  * @example
  * ignoreProperties = {
  *   value1: ["item11", "item12", "item13"],
- *   value2: ["item21", "item22", "item23"],
- *   value3: ["item31", "item32", "item33"],
+ *   value2: "item2",
  * };
  * validateObjectWithArrayProps(isString)(ignoreProperties);
  * //=> true
  *
- * @template {(value: unknown) => boolean} Validator
- * @param {Validator | Validator[]} validator
- * @returns {(value: unknown) => boolean}
+ * @typedef {(value: unknown) => boolean} Validator
+ * @param {...Validator} validators
+ * @returns {Validator}
  */
-module.exports = function validateObjectWithArrayProps(validator) {
+module.exports = function validateObjectWithArrayProps(...validators) {
```

Comment:
[note] I've changed the argument type for a more straightforward implementation. However, I think it's safe this utility has not been exposed.